### PR TITLE
Update URLs to the generated documentation with `latest/`

### PR DESCRIPTION
**Pull request by gctucker**
Update links to the URLs with the generated documentation when running the API on localhost and staging with the API version prefix `latest/` added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,11 +39,12 @@ stages.
...
-To use the generated documentation, first open it on
-http://localhost:8001/docs.  Then click on "Authorize", enter the user name and
-password and click on the "Authorize" button.
+To use the generated documentation, first [open
+it](http://localhost:8001/latest/docs) with a web browser.  Then click on
+"Authorize 🔓", enter the user name and password and click on the new
+"Authorize" button.
 
 This is based on OpenAPI, and you can also download the `openapi.json` file to
 use it with other tools: http://localhost:8001/openapi.json
```
Comment (JenySadadia) [inline README.md:50]: This should also be updated:
```suggestion
use it with other tools: http://localhost:8001/latest/openapi.json
```

**Comment by gctucker** [comment]
Also there's something strange, I think I've already got these changes merged :thinking:

**Comment by gctucker** [comment]
> Also there's something strange, I think I've already got these changes merged thinking

See #231 

What's strange is that this PR is still showing the diff... taking a closer look.  And yes the `openapi.json` link needs to be fixed anyway.

**Comment by gctucker** [comment]
I think that was just a glitch with a branch I had dangling in my local checkout in `kernelci-project/kernelci.org/external/kernelci-api`.  Anyway, I'll make another PR with the `openapi.json` fix.

**Pull request closed without merging**